src/WithdrawalQueueManager.sol

```diff
@@ -291,6 +291,9 @@ contract WithdrawalQueueManager is IWithdrawalQueueManager, ERC721EnumerableUpgr
 
         uint256 unitOfAccountAmount = calculateRedemptionAmount(request.amount, redemptionRate);
 
+        // zero out the last decimal place
```

Comment:
what is the reason to zero out the last decimal place?

Reply:
removed at https://github.com/yieldnest/yieldnest-protocol/commit/7b35647e5cc94a991b80e5a9144990d0d179f013.